src/lapack/gpu/add.cu

```diff
@@ -0,0 +1,170 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2023, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+#include <whip.hpp>
+
+#include "dlaf/gpu/assert.cu.h"
+#include "dlaf/gpu/blas/api.h"
+#include "dlaf/lapack/gpu/add.h"
+#include "dlaf/types.h"
+#include "dlaf/util_cublas.h"
+#include "dlaf/util_math.h"
+
+namespace dlaf::gpulapack {
+namespace kernels {
+
+using namespace dlaf::util::cuda_operators;
```

Comment:
Does this need to be in namespace scope?

Reply:
It is limited to this source file and it is needed by most of the device functions (1 line vs 5 lines). So why not?

Reply:
Good point, ignore me.